### PR TITLE
Initial elm dev; allow user to input date for total US cases

### PR DESCRIPTION
**Pull request by mjhoy**
Captura de Pantalla 2020-04-12 a la(s) 13 25 14

### CODE REVIEW

**Comment by mjhoy** [comment]
@camenpihor i don't know why i failed the backend tests. is 9.73/10 out of bad?

**Comment by camenpihor** [comment]
image

Yea, we require 10/10. Doesn't like the #TODO which I agree with (like not having TODOs in production code). Thoughts on how to handle CORS, in PR comment

File: backend/api/app.py [hunk 1]
```diff
@@ -48,4 +48,7 @@ def get_confirmed_to_date(date):
 
 def json_response(data):
     """Return a Flask response with the mimetype set to JSON."""
-    return Response(data, mimetype="application/json")
+    resp = Response(data, mimetype="application/json")
+    # TODO: Only add this for development mode?
```
Comment (camenpihor) [inline backend/api/app.py:52]: Should remove TODO, maybe add issue if we can't resolve? Or add TODO to the readme?

File: backend/api/app.py [hunk 2]
```diff
@@ -48,4 +48,7 @@ def get_confirmed_to_date(date):
...
+    resp.headers["Access-Control-Allow-Origin"] = "http://localhost:8000"
```
Comment (camenpihor) [inline backend/api/app.py:53]: We _could_ control this via an environment variable? Kinda makes starting the servers more complex though. 

Hmmmmmmm, yea maybe we just do env variables and we the `scripts/run` scripts can just read those in. What do you think?

Comment (camenpihor) [reply backend/api/app.py:53]: Proposal: 

```.env
BACKEND_ADDRESS = http://localhost:5000
FRONTEND_ADDRESS = http://localhost:8000
```

File: frontend/.gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 elm-stuff
+public/application.js
```
Comment (camenpihor) [inline frontend/.gitignore:2]: Do we want any files from `public/`?

File: frontend/src/Main.elm [hunk 4]
```diff
@@ -1,22 +1,97 @@
 module Main exposing (..)
```
Comment (camenpihor) [inline frontend/src/Main.elm:1]: I _think_ you can just expose `main` here

File: frontend/src/Main.elm [hunk 5]
```diff
@@ -1,22 +1,97 @@
...
 
 import Browser
-import Html exposing (Html, div, h1, text)
+import Html exposing (..)
+import Html.Attributes exposing (..)
```
Comment (camenpihor) [inline frontend/src/Main.elm:5]: Can we not use this syntax? and instead import only the things that we need?

File: backend/api/app.py (hunk 1 above)
Comment (mjhoy) [reply backend/api/app.py]: resolved with #13

File: backend/api/app.py (hunk 2 above)
Comment (mjhoy) [reply backend/api/app.py]: resolved with #13

File: frontend/src/Main.elm (hunk 5 above)
Comment (mjhoy) [reply frontend/src/Main.elm]: it's pretty annoying to have to go up and add `h4` when you want to add an `h4` element, etc. what do you think about 5d5ba1da85f6d0d8057f043fd4d68a86b2f3dd68 ?

File: frontend/src/Main.elm (hunk 4 above)
Comment (mjhoy) [reply frontend/src/Main.elm]: 👍 e2e4ab5c1c4f7296f61b2d76e6074ad581728bee

File: frontend/.gitignore (hunk 3 above)
Comment (mjhoy) [reply frontend/.gitignore]: oops, i don't actually use this directory! fdebee1ac016ef57fff9752e6475bbbc66df7c02

File: frontend/src/Main.elm (hunk 5 above)
Comment (camenpihor) [reply frontend/src/Main.elm]: Hmmmmmmm, I'm _fine_ with that haha. Good compromise!

Comment (mjhoy) [reply frontend/src/Main.elm]: :)

**Pull request merged**